src/overrides.css

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2018-2020 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+/*
+ * This file is for patternfly css specific overrides
+ */
+
+.pf-c-app-launcher__menu-item-external-icon {
```

Comment:
Could we use another CSS selector here
```
*.pf-c-app-launcher__menu-item-external-icon {
  color: var(--pf-global--icon--Color--light);
  opacity: 1;
}
```
 instead of 
```
.pf-c-app-launcher__menu-item-external-icon {
  color: var(--pf-global--icon--Color--light) !important;
  opacity: 1 !important;
}
```
?